pkg/licverifier/verifier.go

```diff
@@ -61,12 +77,18 @@ func NewLicenseVerifier(pemBytes []byte) (*LicenseVerifier, error) {
 
 // Verify verifies the license key and validates the claims present in it.
 func (lv *LicenseVerifier) Verify(license string) (LicenseInfo, error) {
-	var licInfo LicenseInfo
-	_, err := jwt.ParseWithClaims(license, &licInfo, func(token *jwt.Token) (interface{}, error) {
+	var claims licenseClaims
+	_, err := jwt.ParseWithClaims(license, &claims, func(token *jwt.Token) (interface{}, error) {
```

Comment:
ParseWithClaims already validates the token @krisis you should basically validate the token returned by jwt.ParseWithClaims

```suggestion
	token, err := jwt.ParseWithClaims(license, &claims, func(token *jwt.Token) (interface{}, error) {
```

```
if !token.Valid {
}
```

Reply:
Since you don't set SkipClaimsValidation - claims are already validated @krisis

Reply:
https://godoc.org/github.com/dgrijalva/jwt-go#Token.Valid

https://godoc.org/github.com/dgrijalva/jwt-go#Claims.Valid()

So we can avoid external validators.. since its not just ExpiresAt, we need to validate IssuedAt

https://github.com/dgrijalva/jwt-go/blob/master/claims.go#L32